vizro-core/examples/_dev/app.py

```diff
@@ -1,50 +1,67 @@
 """Dev app to try things out."""
 
+import pandas as pd
 import vizro.models as vm
-import vizro.plotly.express as px
 from vizro import Vizro
-from vizro.actions import filter_interaction
-from vizro.models.types import capture
+from vizro.figures import kpi_card, kpi_card_reference
 
-df = px.data.iris()
+df_kpi = pd.DataFrame({"Actual": [100, 200, 700], "Reference": [100, 300, 500], "Category": ["A", "B", "C"]})
 
+example_cards = [
+    kpi_card(data_frame=df_kpi, value_column="Actual", title="KPI with value"),
+    kpi_card(data_frame=df_kpi, value_column="Actual", title="KPI with aggregation", agg_func="median"),
+    kpi_card(
+        data_frame=df_kpi,
+        value_column="Actual",
+        title="KPI with formatting",
+        value_format="${value:.2f}",
+    ),
+    kpi_card(
+        data_frame=df_kpi,
+        value_column="Actual",
+        title="KPI with icon",
+        icon="shopping_cart",
+    ),
+]
 
-@capture("graph")
-def my_graph(data_frame, custom_data=None):
-    """Custom graph function."""
-    return px.scatter(
-        data_frame,
-        title="Title",
-        x="sepal_width",
-        y="sepal_length",
-        color="species",
-        # This does not work
-        custom_data=["species"],
-        # This works
-        # custom_data=custom_data
-    )
-
+example_reference_cards = [
+    kpi_card_reference(
+        data_frame=df_kpi,
+        value_column="Actual",
+        reference_column="Reference",
+        title="KPI reference (pos)",
+    ),
+    kpi_card_reference(
+        data_frame=df_kpi,
+        value_column="Actual",
+        reference_column="Reference",
+        agg_func="median",
+        title="KPI reference (neg)",
+    ),
+    kpi_card_reference(
+        data_frame=df_kpi,
+        value_column="Actual",
+        reference_column="Reference",
+        title="KPI reference with formatting",
+        value_format="{value:.2f}$",
+        reference_format="{delta:.2f}$ vs. last year ({reference:.2f}$)",
```

Comment:
```suggestion
        reference_format="{delta:+.2f}$ vs. last year ({reference:.2f}$)",
```
maybe? Doesn't really matter but would match the default better.